fdbbackup/backup.actor.cpp

```diff
@@ -1112,6 +1114,9 @@ static void printBackupUsage(bool devhelp) {
 	       "and ignore the range files.\n");
 	printf("  --encryption-key-file"
 	       "                 The AES-128-GCM key in the provided file is used for encrypting backup files.\n");
+	printf("  --encrypt-files 0/1"
```

Comment:
Shall we make `--encryption-key-file` and `--encrypt-files` incompatible with each other (return error if both are used) and note that in the help message?

Reply:
I'm pretty sure these are completely separate features and are compatible with one another. Encryption key files encrypts at the file level (for EBS snapshots) while my flag does it at the block level. I don't think there is a reason to exclude one or the other.

Reply:
Yeah, that's what I see from the code that they are parallel. But from a user point of view they will never be used together. Also the help message here is quite confusing for someone without context to know which is which. We shall at least update the help message, and throw warning if not error if they are used together.

Reply:
Changed the wording and added a warning when these are used together